src/Acuminator/Acuminator.Vsix/Commands/DiagnosticSuppression/ProjectSuppressionFileGenerator.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Dynamic;
+using System.Linq;
+using System.Threading;
+using System.IO;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Acuminator.Utilities.Common;
+using Acuminator.Utilities.Roslyn;
+using Acuminator.Utilities.DiagnosticSuppression;
+using Acuminator.Vsix.Utilities;
+
+
+
+namespace Acuminator.Vsix.DiagnosticSuppression
+{
+	/// <summary>
+	/// A generator of supression file for project.
+	/// </summary>
+	public static class ProjectSuppressionFileGenerator
```

Comment:
My opinion is that this class should be neither removed nor moved to Acuminator.Utilities into DiagnosticsSuppression folder